lib/kafkat/command/reassign.rb

```diff
@@ -37,9 +38,17 @@ def run
         broker_count = broker_ids.size
 
         topics.each do |_, t|
+          if opts[:partitions]
```

Comment:
I think the command should fail if the partition list passed as argument contains partitions that do not exist.